src/AppInstallerCLITests/YamlManifest.cpp

```diff
@@ -226,7 +226,9 @@ TEST_CASE("ReadBadManifests", "[ManifestValidation]")
         { "Manifest-Bad-VersionInvalid.yaml", "Invalid field value. Field: Version" },
         { "Manifest-Bad-VersionMissing.yaml", "Required field missing. Field: Version" },
         { "Manifest-Bad-InvalidManifestVersionValue.yaml", "Invalid field value. Field: ManifestVersion" },
-        { "InstallFlowTest_MSStore.yaml", "Field value is not supported. Field: InstallerType Value: MSStore" },

```

Comment:
>Field value is not supported. Field: InstallerType Value: MSStore [](start = 43, length = 65)

Is it intentional to remove this error message? I think it's good and informational to manifest authors

Reply:
I believe the problem comes from the failure in ManifestVersion completely blocking all other validation.  I don't think we can proceed if we don't like the version.  But I can drop the fullValidation from ManifestVer and allow it to get back to the original error here.

---
In reply to: [490678974](https://github.com/microsoft/winget-cli/pull/574#discussion_r490678974) [](ancestors = 490678974)